incubator/redis-exporter/values.yaml

```diff
@@ -0,0 +1,14 @@
+replicaCount: 1
+image:
+  repository: oliver006/redis_exporter
+  tag: v0.13
+  pullPolicy: IfNotPresent
+service:
+  type: ClusterIP
+  externalPort: 9121
+  internalPort: 9121
+  annotations:
```

Comment:
I'd suggest you comment these out. The annotations are not necessary if the Prometheus Operator is used.